### PR TITLE
Refactoring of AstTranslator

### PR DESCRIPTION
**Pull request by b-scholz**
Refactored some code in AstTranslator to remove redundant helper methods, etc.

### CODE REVIEW

File: src/AstTranslator.h [hunk 1]
```diff
@@ -59,27 +58,27 @@ class AstTranslator {
     const TypeEnvironment* typeEnv;
 
     /**
-     * The location of some value in a loop nest.
+     * Concrete attribute
      */
     struct Location {
         int level;         // < the loop level
-        int component;     // < the component within the tuple created in the given level
+        int element;       // < the element within the tuple created in the given level
```
Comment (mmcgr) [inline src/AstTranslator.h]: While you're modifying this, could you remove the '< ' from these three comments? We don't use it anywhere else.

**Comment by souffle-lang** [comment]
[64% (-0.01%) vs master 64%](http://plang1.it.usyd.edu.au/jenkins/job/PR-RHEL7-gcc7-Coverage/266/)

**Comment by souffle-lang** [comment]
[64% (0.0%) vs master 64%](http://plang1.it.usyd.edu.au/jenkins/job/PR-RHEL7-gcc7-Coverage/269/)

**Comment by mmcgr** [comment]
Jenkins, test this please

**Comment by souffle-lang** [comment]
[64% (-0.02%) vs master 64%](http://plang1.it.usyd.edu.au/jenkins/job/PR-RHEL7-gcc7-Coverage/270/)

**Comment by souffle-lang** [comment]
[64% (-0.02%) vs master 64%](http://plang1.it.usyd.edu.au/jenkins/job/PR-RHEL7-gcc7-Coverage/271/)

**Comment by mmcgr** [comment]
Jenkins, test this please

**Comment by souffle-lang** [comment]
[64% (-0.01%) vs master 64%](http://plang1.it.usyd.edu.au/jenkins/job/PR-RHEL7-gcc7-Coverage/272/)

**Pull request merged**